mk/spksrc.common.mk

```diff
@@ -20,6 +20,8 @@ PIP_CACHE_OPT ?= --cache-dir $(PIP_DIR)
 PIP_WHEEL_ARGS = wheel --no-binary :all: $(PIP_CACHE_OPT) --no-deps --requirement $(WORK_DIR)/wheelhouse/requirements.txt --wheel-dir $(WORK_DIR)/wheelhouse -b $(WORK_DIR)/wheelbuild
 PIP_WHEEL = $(PIP) $(PIP_WHEEL_ARGS)
 
+PATH := $(WORK_DIR)/../../../native/nodejs/work-native/node/bin:$(PATH)
+
```

Comment:
As this is specific for `native/nodejs` dependent packages this must not be added to this common makefile.